### PR TITLE
fix: add stage 2 outputs

### PR DESCRIPTION
**Pull request by apeabody**

### CODE REVIEW

File: 2-multitenant/envs/development/outputs.tf [hunk 1]
```diff
@@ -14,7 +14,22 @@
...
+}
+
 output "clusters_ids" {
   description = "GKE cluster IDs"
   value       = module.env.cluster_ids
 }
+
+output "cluster_membership_ids" {
+  description = "GKE cluster IDs"
+  value       = module.env.cluster_ids
```
Comment (yliaog) [inline 2-multitenant/envs/development/outputs.tf:34]: s/cluster_ids/cluster_membership_ids/

File: 2-multitenant/envs/non-production/outputs.tf [hunk 2]
```diff
@@ -14,7 +14,22 @@
...
+}
+
 output "clusters_ids" {
   description = "GKE cluster IDs"
   value       = module.env.cluster_ids
 }
+
+output "cluster_membership_ids" {
+  description = "GKE cluster IDs"
+  value       = module.env.cluster_ids
```
Comment (yliaog) [inline 2-multitenant/envs/non-production/outputs.tf:34]: ditto

File: 2-multitenant/envs/production/outputs.tf [hunk 3]
```diff
@@ -14,7 +14,22 @@
...
+}
+
 output "clusters_ids" {
   description = "GKE cluster IDs"
   value       = module.env.cluster_ids
 }
+
+output "cluster_membership_ids" {
+  description = "GKE cluster IDs"
+  value       = module.env.cluster_ids
```
Comment (yliaog) [inline 2-multitenant/envs/production/outputs.tf:34]: ditto

File: 2-multitenant/envs/development/outputs.tf (hunk 1 above)
Comment (apeabody) [reply 2-multitenant/envs/development/outputs.tf]: Thanks - Incomplete copy/paste. :)

File: 2-multitenant/envs/production/outputs.tf (hunk 3 above)
Comment (apeabody) [reply 2-multitenant/envs/production/outputs.tf]: Thanks - Incomplete copy/paste. :)

**Review by gtsorbo — APPROVED** [review]

**Pull request merged**